doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -177,6 +177,11 @@ every single node property as column prefixed by `source` or `target`
 |`sample`
 |No
 
+|`pushdown.filters.enabled`
```

Comment:
just out of curiosity, why would I ever not want this enabled?   What situation would mean I should disable it?

Reply:
Maybe some users prefer to leave the filtering to Spark because, for instance, Spark would be faster in filtering some properties that might not have been indexed on Neo4j. 

Also, if we have a bug on the filtering, this could be a nice patch 😅